pom.xml

```diff
@@ -388,6 +388,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
                 <version>3.3.0</version>
+                <configuration>
+                    <skip>${shade.skip}</skip>
+                </configuration>
```

Comment:
Yes, sometimes it is helpful for local integration testing. When there is an issue in the tests itself, I do not want to assembly the jar file. 
I am going to remove for now, but in general it would be good to update PK templates with skip parameters later on.